src/productDetails.php

```diff
@@ -0,0 +1,37 @@
+<?php
+include "./components/navbar.php";
+include "./functions/sellerFunctions.php";
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <title>title</title>
+    <link href="https://cdn.jsdelivr.net/npm/daisyui@3.7.4/dist/full.css" rel="stylesheet" type="text/css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+<?php
+if (isset($_GET['gekozenProduct'])) {
+    foreach(getProductInfo($mysqli, $_GET['gekozenProduct']) as $row) {
+      echo '
+      <div class="flex items-center justify-center h-screen">
+        <figure>
```

Comment:
Is het mogelijk om misschien een limiet op de grootte van zo'n afbeelding te zetten? Ik heb het zelf eens getest op de school laptop voorlopig, maar mijn afbeelding bedekt bijna heel het scherm.

Reply:
Ik zal ze nu aanpassen.